.github/workflows/pull-request-checks.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Coverage Check'
+on: [pull_request]
+
+jobs:
+  generate-report:
```

Comment:
wow, really cool! Thanks for doing this.

Btw, where can I find an instruction on how to modify this file?